precompile/stateful_precompile_config.go

```diff
@@ -7,22 +7,34 @@ import (
 	"fmt"
 	"math/big"
 
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/snow/engine/snowman/block"
 	"github.com/ethereum/go-ethereum/common"
 )
 
+// PredicateFunc is the function type for validating that an access list tuple touching a precompile follows the predicate
+type PredicateFunc func(chainContext *snow.Context, blockContext *block.Context, storageSlots []common.Hash) error
```

Comment:
Would it make sense to update the type of storage slots to a byte slice as well?